rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1612,6 +1620,89 @@ private List<Map<String, Object>> getReleaseObjectMapper(List<EntityModel<Releas
 		return modifiedList;
 	}
 
+    public List<Release> filterReleases(User sw360User, String withSourceAttachment, String withoutSourceAttachment, String withCliAttachment, String withAttachment, String withoutAttachment, Set<String> releaseIds) {
```

Comment:
Then you can replace parameters of filterReleases as below

`public List<Release> filterReleases(User sw360User, String filter, Set<String> releaseIds)`